CONTRIBUTING.md

```diff
@@ -12,6 +12,11 @@ and file new ones, all feedback welcome!
 Before you can contribute, you will need to sign the [Contributor License
 Agreement](https://docs.linuxfoundation.org/docs/communitybridge/communitybridge-easycla/contributors).
 
+## Copyright Notices
+
+OpenTelemetry follows [CNCF recommendations](https://github.com/cncf/foundation/blob/master/copyright-notices.md)
```

Comment:
can you mention we are using "OpenTelemetry Authors" as copyright owners. Also please note that we strongly encourage to re-license code under OpenTelemetry when self-authors code is contributed.

Reply:
Done. Also added clarification on how to bring third-party code to comply with CNCF recommendations.